common/wallet/wallet_avm.go

```diff
@@ -0,0 +1,42 @@
+package wallet
+
+import (
+	"crypto/ecdsa"
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+//TODO find which cryptography protocol Algorand uses
+
```

Comment:
It looks like a standard evm wallet, it would be better to reuse existing one (wallet_evm)

Reply:
We haven't looked into the wallets on this PR yet, this is just a placeholder.
But from what I could understand, we might be able to reuse the Keystore decryption algorithm from Near, since the keys are stored in the same format as Algorand in ed25519.